rest/results-tabulation-api/ext/ExtendedElection/ExtendedElectionParliamentaryElection2020/RoleBasedAccess.py

```diff
@@ -22,6 +22,10 @@
             NonPostal: [READ, WRITE, SUBMIT, LOCK],
             Postal: [READ, WRITE, SUBMIT, LOCK]
         },
+        PE_22: {
```

Comment:
There's few places down to add few entries. Check how it's done for PE-4

Reply:
Fixed.